05_stringer/davidbroughsmyth/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
```

Comment:
This is the only thing I could find to comment on... I don't think strconv is required in this case.  It can just be 
`expected := "127.0.0.1\n"
actual := buf.String()`

Reply:
Thanks for the feedback. You are right, I've tried out your suggestion and it works fine.
I will look out for this for future labs.

Reply:
@pfannerj the `strconv.Quote` can still be useful even in these simple cases as it makes unprintable characters printable in the output. Technically `actual` and `expected` to not need to be quoted, and only quoted when printed - you can do this when you do direct comparisons and `Printf` if not using `testify/assert`, but since `assert` will print out the values, having them quoted may help. Consider the case where the wrong format specifier was used - e.g. `%c` instead of `%d` - this may result in unprintable or control characters in the `actual` string, and when printed, are hard to see. Quoting make it readable. I'm making the assumption that `testify` does not do any quoting when outputting the values though. Maybe it does.

Reply:
In some of the other labs had that exact issue when using `assert`, the `strconv.Quote` made the values as printable characters. I also like how clean the code looks when using `assert`.

Reply:
The consensus is then to use` strconv.Quote`. I'll make no changes here.